hummingbot/cli/utils/exchange_rate_conversion.py

```diff
@@ -1,19 +1,24 @@
 import asyncio
 import logging
+import math
 import ujson
 from typing import (
     Optional,
     List,
-    Tuple
-)
-import aiohttp
+    Tuple,
+    Dict)
 from hummingbot.cli.config.global_config_map import global_config_map
+from hummingbot.data_feed.coin_cap_data_feed import CoinCapDataFeed
+from hummingbot.data_feed.data_feed_base import DataFeedBase
+NaN = float("nan")
 
 
 class ExchangeRateConversion:
     erc_logger: Optional[logging.Logger] = None
     _erc_shared_instance: "ExchangeRateConversion" = None
-    _exchange_rate_config_override: Optional[List[Tuple[str, float, str]]] = None
+    _exchange_rate_config_override: Optional[Dict[str, Dict]] = None
+    _price_feeds_override: Optional[List[DataFeedBase]] = None
```

Comment:
Could you call it `data_feed` instead of `price_feed` or change the existing data_feeds to price_feeds? So that we don't have another name standardization task item later on~